### DISCUSSION TITLE
Motion detection by camera, onvif, Motion vs. MotionPlus

### DISCUSSION THREAD

**Discussion by tukey42** [Motion-Project/motion#1677] [Q&A]
I am using Motion for several years and it is in most situation doing a really good job in detecting motions. Nowadays some cameras have better motion detection in some situations. For this reason and to offload the effort to the camera, I like the camera to detect the motion. I am for some cameras using ONVIF and an external script to trigger the motion event in Motion.
Here my questions:
- To disable motion events in "motion" I just set THRESHOLD to 0. Is it better (for the CPU) to use "PAUSE"?
- Is there any plan to implement ONVIF in Motion?
- Is there any good reason to migrate from Motion to MotionPlus today?

Many thanks for your help and the great work!

**Comment by tukey42** [comment]
Still hoping to get some answers on my questions!

**Comment by Mr-Dave** [comment]
Pause is probably lower cpu but you can check on your computer which is better.  
No.
It depends upon your needs.  See [comparison](https://motion-project.github.io/motion_guide.html).  Motion will be stable and unchanging vs Motionplus will be changing and may have more bugs/issues to resolve.

**Comment by McMagnus** [comment]
I've been wanting to do something like that too. Can you give me a pointer to the external script?

**Reply by tukey42** [reply]
Here is a pointer to the script I am using: https://github.com/rostislavzz/motion-onvif-events.
